src/LodeRunner.Core/LodeRunner.Core.xml

```diff
@@ -388,6 +388,11 @@
             The Config Interface.
             </summary>
         </member>
+        <member name="P:LodeRunner.Core.Interfaces.ILRConfig.LoadClientId">
```

Comment:
is this a generated file?  If so, should it be included in git?

Reply:
Agreed.  Please remove the xml docs files (LodeRunner.Core.xml & LodeRunner.API.xml) from Git and and exclude *.xml in .gitignore.  I don't believe that we have any other XML files we need in source.  Check with @gortegaMS if you're unsure.

Reply:
no problem. removed both